src/main/java/bitmap/transformer/App.java

```diff
@@ -11,10 +11,20 @@
 
     public static void main(String[] args) throws IOException {
 
-        Bitmap bitStruc = new Bitmap("/Users/Mattaus/codefellows/401/bitmap-transformer/src/main/resources/mario.bmp");
-        bitStruc.negative();
-
-        bitStruc.writeBitmap("/Users/Mattaus/codefellows/401/test.bmp");
+       // Bitmap bitStruc = new Bitmap("/Users/Mattaus/codefellows/401/bitmap-transformer/src/main/resources/mario.bmp");
+        Bitmap bitStruc = new Bitmap(args[0]);
+        bitStruc.mirror();
```

Comment:
Delete this line - you call the `mirror()` method before going into the switch statement, so the image gets mirrored and then the transform the user actually picked in `args[1]` gets called on top of it.